Formula/cp2k.rb

```diff
@@ -74,12 +67,10 @@ def install
               "-lfftw3", "-lfftw3 -lfftw3_threads"
 
     # Now we build
-    cd "makefiles" do
-      %w[sopt ssmp popt psmp].each do |exe|
-        system "make", "ARCH=Darwin-IntelMacintosh-gfortran", "VERSION=#{exe}"
-        bin.install "../exe/Darwin-IntelMacintosh-gfortran/cp2k.#{exe}"
-        bin.install "../exe/Darwin-IntelMacintosh-gfortran/cp2k_shell.#{exe}"
-      end
+    %w[sopt ssmp popt psmp].each do |exe|
+      system "make", "ARCH=Darwin-IntelMacintosh-gfortran", "VERSION=#{exe}"
+      bin.install "../exe/Darwin-IntelMacintosh-gfortran/cp2k.#{exe}"
+      bin.install "../exe/Darwin-IntelMacintosh-gfortran/cp2k_shell.#{exe}"
```

Comment:
@fxcoudert do these need changing on ARM? It didn't seem to cause any problems before, but it looks like it's causing errors now?

Reply:
No it's called IntelMacintosh for historical reasons but it's actually arch-agnostic. I believe the errors are generic, and we'll see them on Intel once libint compilation is fixed.